com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -220,13 +270,19 @@ static void OnBakeStarted()
             if (!ProbeReferenceVolume.instance.isInitialized) return;
             if (ProbeReferenceVolume.instance.perSceneDataList.Count == 0) return;
 
-            var pvList = GameObject.FindObjectsOfType<ProbeVolume>();
-            if (pvList.Length == 0) return; // We have no probe volumes.
+            var sceneDataList = GetPerSceneDataList();
+            if (sceneDataList.Count == 0) return;
+
+            var pvList = GetProbeVolumeList();
+            if (pvList.Count == 0) return; // We have no probe volumes.
 
             FindWorldBounds(out bool hasFoundInvalidSetup);
             if (hasFoundInvalidSetup) return;
 
-            SetBakingContext(ProbeReferenceVolume.instance.perSceneDataList);
+            SetBakingContext(sceneDataList);
```

Comment:
Here you call SetBakingContext with reduced data list, but in `PerformDilation` you still use the full list, is it expected ?

Reply:
Yes -- after baking is done, all the resolution steps are done on the full list.